ext/pgsql/pgsql.c

```diff
@@ -1767,12 +1780,17 @@ PHP_FUNCTION(pg_fetch_result)
 	}
 	if (field_name) {
 		field_offset = PQfnumber(pgsql_result, ZSTR_VAL(field_name));
+		// TODO Split into 2 and ValueError for negative index?
```

Comment:
No, this wouldn't make sense. Most likely the >= condition cannot even happen here.